### DISCUSSION TITLE
Fresh installation was successful, but could not login

### DISCUSSION THREAD

**Discussion by starworld99** [nextcloud/all-in-one#4384] [Questions] [answered]
Dear experters,

I install aio on my qnap nas devices, the installation was smoothy, nothing wrong. but after the installation, I could not login the startup page, I currently have three passwords, the first is Initial key, I can use this key to finish the installation task, the second is admin key, I think this is the right key to enter into the start page, the third key is backup key.

The question is when I entered the second key into the startup page (my.private.ip.address:11000), nothing happened, after three attampts, the system locked me for a while.  I also tried to edit /var/lib/docker/volume/nextcloud-aio-mastercontainer/_data/data/configration.json, but I cannot login after change.

Any one please help me investigate the issue, thanks in advance.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-can-i-access-nextcloud-locally
